db/migrate/20180614185842_add_new_columns_to_articles.rb

```diff
@@ -0,0 +1,7 @@
+class AddNewColumnsToArticles < ActiveRecord::Migration[5.2]
+  def change
+    add_column :articles, :user_id, :integer
```

Comment:
if articles MUST belong to a user, throwing `null: false` onto this would be a good idea as well